developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_28.rst

```diff
@@ -41,6 +41,11 @@ Added APIs
   * ``files:node:moved``: the node has been moved (and its data is already updated)
   * ``files:node:updated``: the node data has been updated
 
+* ``\OCP\AppFramework\Http\EmptyContentSecurityPolicy::allowEvalWasm(bool)``: sets ``wasm-unsafe-eval`` in ``script-src`` of the Content Security Policy `to allow compilation and execution of WebAssembly on the page <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#unsafe_webassembly_execution>`_
```

Comment:
should be in the backend section around line 120?

Reply:
Right, I copied it from the previous file and pasted it at the wrong place :facepalm: